net/redis/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2007-2015 OpenWrt.org
```

Comment:
@yousong  Actually, since this is code adapted from some older code, you just cannot remove the copyright like this :)

That being said, in the original source, there is no copyright at all.  So you may want to ask @pdf , or just assign copyright to him and add your own below.